jetty-security/src/main/java/org/eclipse/jetty/security/authentication/LoginAuthenticator.java

```diff
@@ -53,8 +61,21 @@ public void prepareRequest(ServletRequest request)
         //empty implementation as the default
     }
 
+    /**
+     * If the UserIdentity is not null after this method calls LoginService.login, it
```

Comment:
Use javadoc's `@{link ... }` to reference `LoginService.login()` method.

Reply:
Done